apps/antalmanac/src/stores/calendarizeHelpers.ts

```diff
@@ -135,6 +137,7 @@ export function calendarizeCustomEvents(currentCustomEvents: RepeatingCustomEven
                 isCustomEvent: true,
                 end: new Date(2018, 0, dayIndex, endHour, endMin),
                 title: customEvent.title,
+                days: days,
```

Comment:
JavaScript shorthand for defining properties with the same name as the variable name.

```suggestion
                days
```

Reply:
First time, and good to know this!